app/helpers/discussions_helper.rb

```diff
@@ -195,4 +195,8 @@ def discussion_filter_params_without_page
   def should_show_approved_for?(user, message)
     !user&.moderator_here? && message.approved? && !message.from_moderator?
   end
+
+  def gendered_moderator(user)
+    user.female? ? t(:female_moderator) : t(:moderator)
+  end
```

Comment:
I'm aware this isn't a good idea on a bigger scale, but it's our first and only (so far) gender-aware translation. We should check out [i18n-inflector](https://github.com/siefca/i18n-inflector) if this necessity grows.